### PR TITLE
Adds a Urban Dictionary command

### LINKED ISSUE #98: Urban Dictionary command
https://www.npmjs.com/package/urban-dictionary

### PR DESCRIPTION
**Pull request by Rodolfoam3**
Closes #98

### CODE REVIEW

**Comment by Doges** [comment]
not fixed, solve your conflicts

**Comment by Doges** [comment]
let me fix that

File: src/locales/en-US/commands.json [hunk 1]
```diff
@@ -347,5 +347,14 @@
     "commandDescription": "Turns your text into ASCII art",
     "commandUsage": "<text>",
     "noText": "You need to give me some text to asciify"
+  },
+  "urban": {
+    "commandDescription": "Searches your word on Urban Dictionary",
```
Comment (pedrofracassi) [inline src/locales/en-US/commands.json]: `Searches for an expression on Urban Dictionary`

File: src/locales/en-US/commands.json [hunk 2]
```diff
@@ -347,5 +347,14 @@
...
+    "commandUsage": "<word>",
```
Comment (pedrofracassi) [inline src/locales/en-US/commands.json]: expression

File: src/locales/en-US/commands.json [hunk 3]
```diff
@@ -347,5 +347,14 @@
...
+    "noWordProvided": "You need to give me a word to search its definition",
+    "definition": "Definition:",
+    "example": "Example:",
+    "noDefinitionFound": "No definition found to {{- text}}"
```
Comment (pedrofracassi) [inline src/locales/en-US/commands.json]: for*

File: src/locales/en-US/commands.json [hunk 4]
```diff
@@ -347,5 +347,14 @@
     "commandDescription": "Turns your text into ASCII art",
     "commandUsage": "<text>",
     "noText": "You need to give me some text to asciify"
+  },
+  "urban": {
+    "commandDescription": "Searches your word on Urban Dictionary",
+    "commandUsage": "<word>",
+    "noWordProvided": "You need to give me a word to search its definition",
```
Comment (pedrofracassi) [inline src/locales/en-US/commands.json]: Only "You need to give me a word" is fine.

File: src/locales/en-US/commands.json [hunk 5]
```diff
@@ -347,5 +347,14 @@
...
+  "urban": {
+    "commandDescription": "Searches for an expression on Urban Dictionary",
+    "commandUsage": "<expression>",
+    "noExpressionProvided": "You need to give me an expression",
+    "definition": "Definition:",
+    "example": "Example:",
+    "noDefinitionFound": "No definition found for {{- text}}"
   }
 }
+}
```
Comment (davipatury) [inline src/locales/en-US/commands.json]: This doesn't exist.

Comment (RodolfoHero) [reply src/locales/en-US/commands.json]: I didn't it wtf

**Comment by pedrofracassi** [comment]
lgtm

**Comment by MrGaabriel** [comment]
This command should check if channel is marked as NSFW, shouldn't?

**Comment by MrGaabriel** [comment]
...sorry for my bad english

**Comment by Doges** [comment]
archiving due 30days of inactivity

**Pull request closed without merging**